src/client/view/container/container.jsx

```diff
@@ -23,34 +25,43 @@ export default class Container extends React.Component {
     }
 
     handleOnSearch(event) {
-        console.log(this.state.searchValue);
-        this.doSearch(this.state.searchValue);
+        if(this.state.searchValue){
+            this.props.doSearch({
+                search: this.state.searchValue,
+                searchBy: this.props.searchBy
+            });
+        }
         event.preventDefault();
     }
 
     handleBackToSearch() {
         this.setState({currentMovie: undefined});
+        this.setState({searchValue: undefined});
+    }
+
+    findMovie = (movies, id) => {
```

Comment:
this.props are available here, there is no sense to add movies as a variable, you may just use `this.props.movies`